### DISCUSSION TITLE
What is the colour palette the photographs use?

### DISCUSSION THREAD

**Discussion by Emerald310** [mortuusars/Exposure#119] [Q&A] [answered]
Hi, just wondering, what is the colour palette that photographs use? In one of the gallery images on modrinth it makes a reference to it being 256 colours. But I'm wondering if there's maybe a 16x16 image out there or something that actually contains all of the specific colours. Just interested because I'm currently exploring the idea of making pixel art images, then importing them into a world using the interplanar projector, but the colours are always changed in unexpected ways. It would be easier to just create the pixel art from the colour palette photos use, if possible. Thanks!

**Comment by mortuusars** [comment, marked as answer]
It's using vanilla [Map Colors](https://minecraft.wiki/w/Map_item_format#Map_colors).
I'm thinking about using separate palette for Exposure just to eliminate the chance of other mods changing map colors in some way (didn't see it happen yet, but mods such as Polytone can change the colors I believe).
Different palettes have also been suggested (or at least talked about), but idk about this - it'll be a bit of a complicated thing to implement.
